src/Input.js

```diff
@@ -93,6 +93,10 @@ class Input extends React.Component {
       attributes.type = type;
     }
 
+    if (type === 'textarea') {
```

Comment:
I think we may want to do something a little more restrictive. It looks like the only input which allows children is `<select>` (and `plaintext`/`staticInput`). Maybe we should do something like
```js
if (!(plaintext || staticInput || type === 'select' || typeof tag !== 'string')) {
  attributes.children = null;
}
```
After though, I also added `typeof tag !== 'string'` as if the user is passing another component to render (via the `tag` prop), we should just let them do what they want as they may render a `<select>` or something else which allows children. (similar use-case as it being in the `if` condition above)

Reply:
I think it should be `typeof tag !== 'string'` since we negate the entire condition.
For instance, if `typeof tag` is "function", then `typeof tag !== 'string'` will be `true`, which would get negated to become `false` which skip the code block and keep children.

It can be rewritten to make it more clear:
```js
if (!plaintext && !staticInput && type !== 'select' && typeof tag === 'string') {
  attributes.children = null;
}
```